src/pages/docs/packaging-applications/build-servers/github-actions.md

```diff
@@ -109,10 +108,9 @@ jobs:
 | `release_number`      | The number for the new Release. If omitted, Octopus Deploy will generate a Release number.                                                           |
 | `environments`        | A list of Environments in Octopus Deploy in which to run (i.e. Dev, Test, Prod). Add each environment on a new line.                                 |
 | `variables`           | A list of Variables to use in the Deployment in `key: value` format. Add each variable on a new line.                                                |
-| `git_ref`             | The Git branch from which to source the project code. Required for Projects using version control in Octopus.                                        |
-| `git_commit`          | The Git commit from which to source the project code. Required for Projects using version control in Octopus.                                        |
-| `server`              | The base URL hosting Octopus Deploy (i.e. `https://octopus.example.app`). It is strongly recommended that this value retrieved from a [GitHub secret](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions). |
-| `service_account_id`  | The id of the service account you wish to login as.                                                                                                  |
+| `git_ref`             | The Git branch from which to source the project code. Required for Projects using version control in Octopus. The example above sources this value from the GitHub Action's [contextual information.](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context) |
+| `git_commit`          | The Git commit from which to source the project code. Required for Projects using version control in Octopus. The example above sources this value from the GitHub Action's [contextual information.](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context) |
```

Comment:
When I came back to use this example code a month later, it wasn't clear to me where the example values were being sourced.